crates/router/src/routes/api_keys.rs

```diff
@@ -190,26 +177,27 @@ pub async fn api_key_update(
 pub async fn api_key_update(
     state: web::Data<AppState>,
     req: HttpRequest,
-    path: web::Path<(common_utils::id_type::MerchantId, String)>,
```

Comment:
can you update this in openapi also for v2?

Reply:
this was updated by you on august 28, 2024 in #5709([api_keys.rs](https://github.com/juspay/hyperswitch/pull/5709/files#diff-7c85aea5f3131c88211bf2b89db7c0dda2aa73ab584ced6cf6f57ff793cec541))